src/components/home/LandingTitle.jsx

```diff
@@ -1,5 +1,25 @@
+import * as React from "react";
+import Icon from "public/HLG-logo.png";
+import Button from "src/components/Button";
+import Image from "next/image";
+
+console.log(Icon);
+
 const LandingTitle = () => {
-  return <div>LandingTitle</div>;
+  return (
+    <div className="flex items-center">
+      <Image src={Icon} alt="Icon" className="w-1/12" />
+      <div className="flex-col">
```

Comment:
add some padding top to this \<div className="flex-col"\> so it looks more like the design doc https://www.figma.com/file/AF6AdslHuzkzUe6qDgcJx9